Package.swift

```diff
@@ -22,7 +22,7 @@ let package = Package(
     // Dependencies declare other packages that this package depends on.
     // .package(url: /* package url */, from: "1.0.0"),
     .package(url: "https://github.com/google/swift-benchmark.git", .branch("master")),
-    .package(url: "https://github.com/saeta/penguin.git", .branch("master")),
+    .package(url: "https://github.com/saeta/penguin.git", .branch("specialized-anyarray")),
```

Comment:
reminder to update to `master` after merging https://github.com/saeta/penguin/pull/97, before merging this